Automattic/wp-calypso
# Refactor help-center components and add chat history feature

| Recent Conversations  | Support History |
| ------------- | ------------- |
| <img width="651" alt="Screenshot 2024-10-18 at 16 08 39" src="https://github.com/user-attachments/assets/0e1c8fc8-1018-4438-ae65-e6ccc9fe80ef"> | <img width="816" alt="Screenshot 2024-10-18 at 16 08 32" src="https://github.com/user-attachments/assets/c1518b97-9817-4439-b203-12e29d917300"> |

Fixes https://github.com/Automattic/dotcom-forge/issues/9275
Display recent conversations and support history.
This work is behind feature flag `help-center-experience`

## Proposed Changes

* Check for unread conversations and messages. 
* Display recent conversations that lead to support history route
* Display conversations in support history and open conversation on click

## Why are these changes being made?


*

## Testing Instructions
- Checkout this PR and build it locally
- Add the ff in the url `?flags=help-center-experience`
- Create a conversation and type a message
- Locate the relevant ticket in the support sandbox
- Assign to yourself and reply. This will create an unread message in the conversation
- Checkout how that is rendered
- Test various conditions here, with the read/unread count
- Clicking on a recent conversation should navigate to support-history where all chats are displayed
- Clicking on a chat there should open the conversation and allow you to reply



*
